framework_tvb/tvb/tests/framework/adapters/testadapter1.py

```diff
@@ -36,11 +36,24 @@
 
 import tvb.core.adapters.abcadapter as abcadapter
 from tvb.basic.neotraits.api import Attr
-from tvb.core.neotraits.forms import SimpleIntField, TraitDataTypeSelectField
+from tvb.core.neotraits.forms import SimpleIntField, TraitDataTypeSelectField, IntField
+from tvb.core.neotraits.view_model import ViewModel
 from tvb.tests.framework.datatypes.datatype1 import Datatype1
 from tvb.tests.framework.datatypes.dummy_datatype import DummyDataType
 from tvb.tests.framework.datatypes.dummy_datatype_index import DummyDataTypeIndex
 
+class TestModel(ViewModel):
+
+    test1_val1 = Attr(
+        field_type=str,
+        default='test1_val1'
```

Comment:
You could use directly Str(default='test1_val1') from view_model.py instead of Attr(field_type=str,default='test1_val1')